rnd/autogpt_server/autogpt_server/data/block.py

```diff
@@ -36,6 +38,7 @@ class BlockCategory(Enum):
     INPUT = "Block that interacts with input of the graph."
     OUTPUT = "Block that interacts with output of the graph."
     LOGIC = "Programming logic to control the flow of your agent"
+    DEVELOPER_TOOLS = "Developer tools like Github."  # added this so all the github blocks are in the same category for now, this is to be changed
```

Comment:
what needs to be changed? The desc or the name? I quite like the category name tbh

Reply:
@Bentlybro

Reply:
It was a suggestion to maybe change the name or description if needed but if its fine and does not need changing then the comment can be removed and it can be kept as is 😄